src/test/functional/features/joint-rfi-response.feature

```diff
@@ -0,0 +1,249 @@
+Feature: Joint request for information
+
+  Background: Logged in for hub page
+    Given I create a new user and login
+    And I've already completed the form using the fixture "jointApplicant1CompleteCase"
+    When I go to "/"
+    And I click "Send for review"
+    Then the page URL should be "/application-sent-for-review"
+    When I enter my valid case reference and valid access code
+    Then the page should include "You need to review your joint application"
+    Given I've already completed the form using the fixture "jointApplicant2CompleteCase" for "applicant2"
+    And I go to "/applicant2/confirm-your-joint-application"
+    Given I select "I confirm that I’m applying to the court to dissolve my marriage (get a divorce)"
+    And I select "I believe that the facts stated in this application are true"
+    When I click "Submit"
+    Then the page URL should be "/applicant2/needs-to-confirm-joint-application"
+
+  @nightly
+  Scenario: [1] Applicant 1 joint rfi journey
```

Comment:
Do you think we should make the happy path run outside of nightly to reduce the chance of people introducing RFI bugs by only relying on normal CI?

Reply:
Yes - I have removed the @nightly decorator from the sole applicant and joint app1 and 2 happy paths